iroh/Cargo.toml

```diff
@@ -70,12 +70,14 @@ regex = { version = "1.7.1", features = ["std"] }
 testdir = "0.8"
 tokio = { version = "1", features = ["macros", "io-util", "rt"] }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
-iroh-io = { version = "0.2.1" }
 postcard = "1"
 proptest = "1.2.0"
 tempfile = "3.4"
 genawaiter = { version = "0.99", features = ["futures03"] }
 
+[profile.release]
```

Comment:
I don't think this needs to be here, I think only the ones in the top level are picked up